modules/references/tests/client/components/CreateReference.client.component.test.js

```diff
@@ -0,0 +1,147 @@
+import React from 'react';
+import {
+  render,
+  fireEvent,
+  waitForElement,
+  waitForElementToBeRemoved,
+  screen
+} from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+
+import * as references from '@/modules/references/client/api/references.api';
+const api = { references };
+
+jest.mock('@/modules/references/client/api/references.api');
+afterEach(() => jest.clearAllMocks());
+
+import CreateReference from '@/modules/references/client/components/CreateReference.component';
```

Comment:
Nit; I think it would be nice to group imports on top before any other code as a convention? I see benefit of having related imports+code grouped together, too, but I'd lean towards grouping imports anyway.

Not a biggie either way.

Reply:
Agree, even better we can add https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/first.md and https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/order.md

Reply:
I added extending from `plugin:import/errors` and upgrading `import/first` to an error. In the future we can tighten it up with `import/order` too, but that makes too many unrelated changes for this PR.